<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
<!DOCTYPE bugzilla SYSTEM "https://bugzilla.redhat.com/page.cgi?id=bugzilla.dtd">

<bugzilla version="4.4.12068.1"
          urlbase="https://bugzilla.redhat.com/"
          
          maintainer="bugzilla-error-list@redhat.com"
>

    <bug>
          <bug_id>1379180</bug_id>
          
          <creation_ts>2016-09-25 15:45:00 -0400</creation_ts>
          <short_desc>Please update to 0.4.5</short_desc>
          <delta_ts>2016-10-14 01:01:15 -0400</delta_ts>
          <reporter_accessible>1</reporter_accessible>
          <cclist_accessible>1</cclist_accessible>
          <classification_id>2</classification_id>
          <classification>Fedora</classification>
          <product>Fedora</product>
          <component>python-django-tagging</component>
          <version>24</version>
          <rep_platform>Unspecified</rep_platform>
          <op_sys>Unspecified</op_sys>
          <bug_status>CLOSED</bug_status>
          <resolution>ERRATA</resolution>
          
          
          <bug_file_loc></bug_file_loc>
          <status_whiteboard></status_whiteboard>
          <keywords>Reopened</keywords>
          <priority>unspecified</priority>
          <bug_severity>unspecified</bug_severity>
          <target_milestone>---</target_milestone>
          
          
          <everconfirmed>1</everconfirmed>
          <reporter name="Piotr Popieluch">piotr1212</reporter>
          <assigned_to name="Matthias Runge">mrunge</assigned_to>
          <cc>jdornak</cc>
    
    
    <cc>mrunge</cc>
          <qa_contact name="Fedora Extras Quality Assurance">extras-qa</qa_contact>
          
          <cf_fixed_in>python-django-tagging-0.4.5-1.fc26 python-django-tagging-0.4.5-1.fc24 python-django-tagging-0.4.5-1.fc25</cf_fixed_in>
          <cf_doc_type>If docs needed, set a value</cf_doc_type>
          <cf_release_notes></cf_release_notes>
          <cf_story_points>---</cf_story_points>
          
          <cf_environment></cf_environment>
          <cf_last_closed>2016-10-10 17:50:08</cf_last_closed>
          <cf_type>Bug</cf_type>
          <cf_regression_status>---</cf_regression_status>
          <cf_mount_type>---</cf_mount_type>
          <cf_documentation_action>---</cf_documentation_action>
          <cf_crm></cf_crm>
          <cf_verified_branch></cf_verified_branch>
          <cf_category>---</cf_category>
          <cf_ovirt_team>---</cf_ovirt_team>
          
          <cf_cloudforms_team>---</cf_cloudforms_team>
          
          
          
          
          <target_release>---</target_release>
          
          <votes>0</votes>

      

      

      

          <comment_sort_order>oldest_to_newest</comment_sort_order>  
          <long_desc isprivate="0" >
    <commentid>9741036</commentid>
    <comment_count>0</comment_count>
      <attachid>1204629</attachid>
    <who name="Piotr Popieluch">piotr1212</who>
    <bug_when>2016-09-25 15:45:05 -0400</bug_when>
    <thetext>Created attachment 1204629
update spec to 0.4.5

I&apos;m updating the graphite-web package in Fedora branches get it working again on Django 1.9 but now I&apos;m getting the following error while running graphite wsgi:

InvalidTemplateLibrary: Invalid template library specified. ImportError raised when trying to load &apos;tagging.templatetags.tagging_tags&apos;: cannot import name get_model

Updating to django-taggin 0.4.5 seems to fix the issue, also described here: 
https://github.com/django-extensions/django-extensions/issues/705
https://github.com/graphite-project/graphite-web/issues/1538

I&apos;ve attached the diff of the updated django-tagging spec. Could you please update django-tagging on all branches with django 1.9 or higher? 
I&apos;m willing to co-maintain and update myself if you prefer (already requested acl).</thetext>
  </long_desc><long_desc isprivate="0" >
    <commentid>9741541</commentid>
    <comment_count>1</comment_count>
    <who name="Matthias Runge">mrunge</who>
    <bug_when>2016-09-26 02:31:10 -0400</bug_when>
    <thetext>done. Also pushed for f25</thetext>
  </long_desc><long_desc isprivate="0" >
    <commentid>9743564</commentid>
    <comment_count>2</comment_count>
    <who name="Piotr Popieluch">piotr1212</who>
    <bug_when>2016-09-26 14:10:43 -0400</bug_when>
    <thetext>Could you also please build for F24? This issue occurs on Django 1.9 or higher which is F24, F25 and Rawhide.</thetext>
  </long_desc><long_desc isprivate="0" >
    <commentid>9745203</commentid>
    <comment_count>3</comment_count>
    <who name="Fedora Update System">updates</who>
    <bug_when>2016-09-27 03:22:05 -0400</bug_when>
    <thetext>python-django-tagging-0.4.5-1.fc25 has been pushed to the Fedora 25 testing repository. If problems still persist, please make note of it in this bug report.
See https://fedoraproject.org/wiki/QA:Updates_Testing for
instructions on how to install test updates.
You can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2016-62ecf3c884</thetext>
  </long_desc><long_desc isprivate="0" >
    <commentid>9749156</commentid>
    <comment_count>4</comment_count>
    <who name="Fedora Update System">updates</who>
    <bug_when>2016-09-28 01:58:11 -0400</bug_when>
    <thetext>python-django-tagging-0.4.5-1.fc24 has been pushed to the Fedora 24 testing repository. If problems still persist, please make note of it in this bug report.
See https://fedoraproject.org/wiki/QA:Updates_Testing for
instructions on how to install test updates.
You can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2016-ecf4d1c5e6</thetext>
  </long_desc><long_desc isprivate="0" >
    <commentid>9779690</commentid>
    <comment_count>5</comment_count>
    <who name="Fedora Update System">updates</who>
    <bug_when>2016-10-10 17:50:08 -0400</bug_when>
    <thetext>python-django-tagging-0.4.5-1.fc24 has been pushed to the Fedora 24 stable repository. If problems still persist, please make note of it in this bug report.</thetext>
  </long_desc><long_desc isprivate="0" >
    <commentid>9790532</commentid>
    <comment_count>6</comment_count>
    <who name="Fedora Update System">updates</who>
    <bug_when>2016-10-14 01:01:15 -0400</bug_when>
    <thetext>python-django-tagging-0.4.5-1.fc25 has been pushed to the Fedora 25 stable repository. If problems still persist, please make note of it in this bug report.</thetext>
  </long_desc>
      
          <attachment
              isobsolete="0"
              ispatch="1"
              isprivate="0"
          >
            <attachid>1204629</attachid>
            <date>2016-09-25 15:45:00 -0400</date>
            <delta_ts>2016-09-25 15:45:05 -0400</delta_ts>
            <desc>update spec to 0.4.5</desc>
            <filename>python-django-tagging.spec.diff</filename>
            <type>text/plain</type>
            <size>1572</size>
            <attacher name="Piotr Popieluch">piotr1212</attacher>
            
              <data encoding="base64">ZGlmZiAtLWdpdCBhL3B5dGhvbi1kamFuZ28tdGFnZ2luZy5zcGVjIGIvcHl0aG9uLWRqYW5nby10
YWdnaW5nLnNwZWMKaW5kZXggOGU0NjA3NC4uNGZiMGNjOCAxMDA2NDQKLS0tIGEvcHl0aG9uLWRq
YW5nby10YWdnaW5nLnNwZWMKKysrIGIvcHl0aG9uLWRqYW5nby10YWdnaW5nLnNwZWMKQEAgLTIs
MTQgKzIsMTQgQEAKIAogJWdsb2JhbCBwa2duYW1lIGRqYW5nby10YWdnaW5nCiBOYW1lOiAgICAg
ICAgICAgcHl0aG9uLWRqYW5nby10YWdnaW5nCi1WZXJzaW9uOiAgICAgICAgMC4zLjYKLVJlbGVh
c2U6ICAgICAgICA1JXs/ZGlzdH0KK1ZlcnNpb246ICAgICAgICAwLjQuNQorUmVsZWFzZTogICAg
ICAgIDElez9kaXN0fQogU3VtbWFyeTogICAgICAgIEEgZ2VuZXJpYyB0YWdnaW5nIGFwcGxpY2F0
aW9uIGZvciBEamFuZ28gcHJvamVjdHMKIAogR3JvdXA6ICAgICAgICAgIERldmVsb3BtZW50L0xh
bmd1YWdlcwogTGljZW5zZTogICAgICAgIE1JVAogVVJMOiAgICAgICAgICAgIGh0dHBzOi8vZ2l0
aHViLmNvbS9GYW50b21hczQyL2RqYW5nby10YWdnaW5nLwotU291cmNlMDogICAgICAgIGh0dHA6
Ly9weXBpLnB5dGhvbi5vcmcvcGFja2FnZXMvc291cmNlL2QvJXtwa2duYW1lfS8le3BrZ25hbWV9
LSV7dmVyc2lvbn0udGFyLmd6CitTb3VyY2UwOiAgICAgICAgaHR0cHM6Ly9maWxlcy5weXRob25o
b3N0ZWQub3JnL3BhY2thZ2VzL3NvdXJjZS9kLyV7cGtnbmFtZX0vJXtwa2duYW1lfS0le3ZlcnNp
b259LnRhci5negogCiBCdWlsZEFyY2g6ICAgICAgbm9hcmNoCiBCdWlsZFJlcXVpcmVzOiAgcHl0
aG9uMi1kZXZlbApAQCAtNzEsMTkgKzcxLDIyIEBAIHBvcGQKIAogCiAlZmlsZXMKLSVkb2MgQ0hB
TkdFTE9HLnR4dCBMSUNFTlNFLnR4dCBSRUFETUUudHh0IGRvY3MvKgorJWRvYyBDSEFOR0VMT0cu
dHh0IExJQ0VOU0UudHh0IFJFQURNRS5yc3QgZG9jcy8qCiAle3B5dGhvbl9zaXRlbGlifS90YWdn
aW5nCiAle3B5dGhvbl9zaXRlbGlifS9kamFuZ29fdGFnZ2luZy0le3ZlcnNpb259LXB5Py4/LmVn
Zy1pbmZvCiAKICVpZiAwJXs/d2l0aF9weXRob24zfQogJWZpbGVzIC1uIHB5dGhvbjMtZGphbmdv
LXRhZ2dpbmcKLSVkb2MgQ0hBTkdFTE9HLnR4dCBMSUNFTlNFLnR4dCBSRUFETUUudHh0IGRvY3Mv
KgorJWRvYyBDSEFOR0VMT0cudHh0IExJQ0VOU0UudHh0IFJFQURNRS5yc3QgZG9jcy8qCiAle3B5
dGhvbjNfc2l0ZWxpYn0vdGFnZ2luZwogJXtweXRob24zX3NpdGVsaWJ9L2RqYW5nb190YWdnaW5n
LSV7dmVyc2lvbn0tcHk/Lj8uZWdnLWluZm8KICVlbmRpZgogCiAKICVjaGFuZ2Vsb2cKKyogU3Vu
IFNlcCAyNSAyMDE2IFBpb3RyIFBvcGllbHVjaCA8cGlvdHIxMjEyQGdtYWlsLmNvbT4gLSAtIDAu
NC41LTEKKy0gVXBkYXRlIHRvIDAuNC41CisKICogVHVlIEp1bCAxOSAyMDE2IEZlZG9yYSBSZWxl
YXNlIEVuZ2luZWVyaW5nIDxyZWwtZW5nQGxpc3RzLmZlZG9yYXByb2plY3Qub3JnPiAtIDAuMy42
LTUKIC0gaHR0cHM6Ly9mZWRvcmFwcm9qZWN0Lm9yZy93aWtpL0NoYW5nZXMvQXV0b21hdGljX1By
b3ZpZGVzX2Zvcl9QeXRob25fUlBNX1BhY2thZ2VzCiAK
</data>

          </attachment>
      

    </bug>

</bugzilla>